README.md

```diff
@@ -150,6 +175,7 @@ func someFunc(client kubernetes.Interface) error {
         MinRetryTime:  time.Millisecond * 100,
         MaxRetryTime:  time.Second * 30,
         MaxRetryCount: 12,
+        UpdateUsername: true
```

Comment:
We should really make not required bool arguments pointers and evaluate nil somewhere,
for now, could we do that just for `UpdateUsername` so we avoid setting it to true everywhere?
We can probably set it once here:
https://github.com/keikoproj/aws-auth/blob/26ae15130a7778f93d49d137ec26ecc8fb712d19/pkg/mapper/types.go#L96-L105



e.g.
```go
var (
    UpdateUsernameArgumentTrue bool = true
)

if args.UpdateUsername == nil {
  args.UpdateUsername = &UpdateUsernameArgumentTrue
}
```

This helps us primarily in the case of using aws-auth as a library, otherwise CLI args set the default to true for us.

Reply:
Bool cant be evaluated against nil. It has a default value of false.  So it looks like this is a breaking change if aws-auth is used as a library. 

```
go run main.go
(main.MapperArguments) {
 KubeconfigPath: (string) "",
 MapRoles: (bool) true,
 MapUsers: (bool) false,
 Force: (bool) false,
[main.go.txt](https://github.com/keikoproj/aws-auth/files/8995252/main.go.txt)

 Username: (string) (len=33) "system:node:{{EC2PrivateDNSName}}",
 RoleARN: (string) (len=78) "arn:aws:iam::555555555555:role/my-new-node-group-NodeInstanceRole-74RF4UBDUKL6",
 UserARN: (string) "",
 Groups: ([]string) (len=2 cap=2) {
  (string) (len=20) "system:bootstrappers",
  (string) (len=12) "system:nodes"
 },
 WithRetries: (bool) true,
 MaxRetryCount: (int) 12,
 IsGlobal: (bool) false,
 Append: (bool) false,
 UpdateUsername: (bool) false
}
```

Reply:
never mind I see what you are asking.